### PR TITLE
data: ShubhamMore6688

### PR DESCRIPTION
**Pull request by ShubhamMore6688**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5662"></a>

### CODE REVIEW

File: data/ShubhamMore6688.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
+  "name": "Shubham More",
+  "type": "personal",
+  "bio": "Devops | Open Source | Android developer ",
+  "avatar": "https://github.com/ShubhamMore6688.png",
```
Comment (imbeshat) [inline data/ShubhamMore6688.json:5]:
```suggestion
```

**Review by imbeshat — CHANGES_REQUESTED** [review]
```avatar``` is not required.

**Review by khushalarora — APPROVED** [review]
Looks good, thanks :tada:

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks for your Contribution! 🎉 

<p align="center"></p>

**Pull request merged**